proptest/src/test_runner/rng.rs

```diff
@@ -435,14 +387,14 @@ impl TestRng {
                 rng: match algorithm {
                     RngAlgorithm::XorShift => {
                         let rng = match seed {
-                            RngSeed::Random => XorShiftRng::from_entropy(),
+                            RngSeed::Random => XorShiftRng::seed_from_u64(0),
```

Comment:
why did you move away from `from_entropy` here and below?

Reply:
Apparently missed it when reviewing https://github.com/proptest-rs/proptest/pull/552.

Fixed it now. Had to pull in the `rand/os_rng` crate feature with our `std` crate feature.